### ISSUE TITLE
Player link in forum causes argument invalid error

### ISSUE THREAD

**Issue by dwvanstone** [buttonmen-dev/buttonmen#2464]
If I create this player link in the forum:

   [player=MaybeYOU?]

I get this error when I click on that forum link:

"Error from loadProfileInfo: Argument (playerName) to function loadProfileInfo is invalid"

It would be nice if it, instead, said "Error from loadProfileInfo: Player name does not exist." or even "Player name does not exist."

**Comment by AdmiralJota** [comment]
Hmm. I'm guessing your objection is that it's a more technical error, rather than a user-friendly one?

If so, might I suggest a compromise, such as "Error from loadProfileInfo: Player name is invalid."?

That would keep the information about *why* the player name was rejected (i.e., not only is it not the name of an existing player, but it's not even a valid name for a player to have), while presenting it in a more natural way, analogous to the message you get when you use a valid but incorrect player name.

**Comment by dwvanstone** [comment]
> Hmm. I'm guessing your objection is that it's a more technical error, rather than a user-friendly one?

Yes - it looks like it's a bug on the site rather than a helpful user error. I like your compromise.
